### ISSUE TITLE
We probably shouldn't have the .db file in the repo

### ISSUE THREAD

**Issue by kashev** [daranguiz/barbell-io#5]
this is not what git is for

We can host a development dataset in [github-lfs](https://git-lfs.github.com/) if we think that this is necessary

**Comment by daranguiz** [comment]
agreed but `git add .` doesn't discriminate

**Comment by bradysalz** [comment]
Is it even MB of data? Who cares. Put it in /tmp as an example

**Comment by daranguiz** [comment]
shrug this is indicative of a more systemic problem -- namely, I don't know how to deal with things like this in git.

Database? Secret keys? Where do all those things live?

**Comment by bradysalz** [comment]
`config.py` which is fine for now. Ideally it would just be a list or dict of key-values, but @kashev is arguing about what style rug you should have when you don't even have a roof.

**Comment by daranguiz** [comment]
Right, but I guess, you just store that stuff locally then? Super weird. Seems like there's gotta be a better way to handle that. 

And meh that's how @kashev rolls. It's easier to keep a project clean if you start it clean.
